src/hooks/useApproveConfirmTransaction.tsx

```diff
@@ -1,5 +1,5 @@
-import React, { useEffect, useReducer, useRef } from 'react'
-import noop from 'lodash/noop'
+import React, { useEffect, useReducer, useRef, useCallback } from 'react'
+import { noop } from 'lodash'
```

Comment:
Import path

Reply:
Oops. It seems prettier auto updates this part. Will check

Reply:
Oops. Noted